### PR TITLE
Fix links via lint

### PR DESCRIPTION
**Pull request by ybiquitous**
`yarn run lint`

```
README.md: no issues found
docs/advanced-settings/inline-comments.md: no issues found
docs/advanced-settings/private-dependencies.md: no issues found
docs/billing-and-plans.md: no issues found
docs/enterprise/quick-start/setup.md: no issues found
docs/enterprise/quick-start/update.md: no issues found
docs/enterprise/releases/changelog.md: no issues found
docs/enterprise/resources.md: no issues found
docs/enterprise/testing/guide.md: no issues found
docs/getting-started/custom-configuration.md: no issues found
docs/getting-started/permissions.md: no issues found
docs/getting-started/repository-settings.md: no issues found
docs/getting-started/setup.md: no issues found
docs/getting-started/working-with-issues.md: no issues found
docs/introduction.md: no issues found
docs/tools/css/scss-lint.md: no issues found
docs/tools/css/stylelint.md: no issues found
docs/tools/go/golint.md: no issues found
docs/tools/go/gometalinter.md: no issues found
docs/tools/go/govet.md: no issues found
docs/tools/java/checkstyle.md: no issues found
docs/tools/java/pmd.md: no issues found
docs/tools/javascript/coffeelint.md: no issues found
docs/tools/javascript/eslint.md: no issues found
docs/tools/javascript/jshint.md: no issues found
docs/tools/javascript/tslint.md: no issues found
docs/tools/others/goodcheck.md: no issues found
docs/tools/others/misspell.md: no issues found
docs/tools/php/codesniffer.md: no issues found
docs/tools/php/phinder.md: no issues found
docs/tools/php/phpmd.md: no issues found
docs/tools/python/flake8.md: no issues found
docs/tools/ruby/brakeman.md: no issues found
docs/tools/ruby/haml-lint.md: no issues found
docs/tools/ruby/querly.md: no issues found
docs/tools/ruby/rails-bestpractices.md: no issues found
docs/tools/ruby/reek.md: no issues found
docs/tools/ruby/rubocop.md: no issues found
docs/tools/swift/swiftlint.md: no issues found
docs/troubleshooting.md: no issues found
website/README.md: no issues found
```

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -27,108 +27,108 @@ This section has listed documents about analysis tools which Sider supports. We
...
-  * [RuboCop](./tools/ruby/rubocop)
-  * [Reek](./tools/ruby/reek)
-  * [Querly](./tools/ruby/querly)
-  * [Rails Best Practices](./tools/ruby/rails-bestpractices)
-  * [Brakeman](./tools/ruby/brakeman)
-  * [HAML-Lint](./tools/ruby/haml-lint)
+  * [RuboCop](./tools/ruby/rubocop.md)
+  * [Reek](./tools/ruby/reek.md)
+  * [Querly](./tools/ruby/querly.md)
+  * [Rails Best Practices](./tools/ruby/rails-bestpractices.md)
```
Comment (ybiquitous) [inline docs/introduction.md]: > __[Goodcheck]__ Is this a typo for "Rails Best Practices"? ([link](https://sider.review/gh/repos/142577083/pulls/103#issue-3664268))

false positive.

**Comment by ybiquitous** [comment]
It works both on GitHub and Netflix!

@akito19 Would you setup lint on CI or release flow?

**Comment by akito19** [comment]
> GitHub and Netflix!

:octocat:  🎥  

> Would you setup lint on CI or release flow?

Ok, I'll do it.

**Comment by ybiquitous** [comment]
Oh, Netflix vs Netlify 😅 It's very confusing! 💢

**Comment by ybiquitous** [comment]
> Ok, I'll do it.

Thank you!

**Pull request merged**